toolkits/springer/packages/springer-header/images/logo.svg

```diff
@@ -0,0 +1 @@
789v-.78c1.478-.164 1.601-.287 1.601-1.684V15.793c0-1.437-.041-1.478-1.437-1.601v-.698c1.232-.246 2.381-.616 3.572-1.027v1.56l2.217-1.15c.287-.164.575-.205.862-.205 2.546 0 3.983 2.217 3.983 4.352 0 3.45-2.463 5.79-5.543 6.16zm.534-8.993c-.657 0-1.519.452-2.053.78v6.077c.452.493 1.191.862 2.094.862 1.191 0 2.751-.903 2.751-3.982 0-2.587-1.232-3.737-2.792-3.737zm12.81 1.027c-.246-.329-.575-.411-.985-.411-.534 0-1.191.616-1.643 1.889v3.819c0 1.396.164 1.52 1.93 1.643v.821h-5.584v-.821c1.396-.123 1.561-.247 1.561-1.643v-4.64c0-1.396-.123-1.396-1.396-1.643v-.698c1.232-.247 2.423-.493 3.49-.903v2.464h.041c.739-1.355 1.766-2.422 2.71-2.422.822 0 1.396.616 1.396 1.273 0 .943-.985 1.765-1.52 1.272zm7.844 7.761h-5.256v-.821c1.396-.123 1.601-.247 1.601-1.724v-4.435c0-1.601-.123-1.601-1.478-1.766v-.698c1.273-.247 2.545-.534 3.572-.903v7.802c0 1.478.123 1.601 1.561 1.724v.821zm-2.711-12.524c-.78 0-1.396-.616-1.396-1.396 0-.821.616-1.437 1.396-1.437.822 0 1.437.616 1.437 1.437 0 .78-.615 1.396-1.437 1.396zm15.275 12.524h-4.845v-.821c1.15-.123 1.314-.247 1.314-1.561v-4.024c0-1.478-.657-2.382-1.889-2.382-.78 0-1.684.575-2.3 1.15v5.256c0 1.314.124 1.438 1.273 1.561v.821h-5.01v-.821c1.396-.123 1.601-.247 1.601-1.561v-4.722c0-1.396-.082-1.478-1.314-1.643v-.698c1.232-.247 2.382-.493 3.449-.903v1.643c.534-.452 2.094-1.601 3.285-1.601 1.848 0 2.997 1.273 2.997 3.162v4.763c0 1.314.206 1.438 1.438 1.561v.82h.001zm11.702-9.773c-.164.369-.616 1.068-.985 1.273l-1.191-.123c.329.452.575 1.355.575 2.053 0 2.505-2.258 3.86-4.311 3.86-.123 0-.452-.041-.739-.041-.493.164-.903.739-.903 1.108 0 .37.37.822 1.355.822H89.6c1.437 0 3.367.698 3.367 2.997 0 2.628-2.751 4.845-5.954 4.845-2.792 0-4.271-1.724-4.271-3.121 0-.903.452-1.601 2.382-3.08-.698-.246-1.93-.944-1.848-2.258.945-.41 1.807-1.15 2.176-1.601-1.108-.37-2.135-1.684-2.135-3.161 0-2.669 2.382-4.106 4.352-4.106a5.1 5.1 0 012.382.575c1.068-.041 2.135-.164 3.121-.287l.163.245zm-3.243 11.005c-.575-.206-1.355-.247-2.628-.247-1.766 0-2.546 1.314-2.546 2.3 0 1.396 1.232 2.464 3.08 2.464 2.053 0 3.203-1.355 3.203-2.71-.001-.863-.494-1.561-1.109-1.807zm-2.628-10.553c-.862 0-1.848.903-1.848 2.628s.903 2.957 2.135 2.957c.903 0 1.766-.78 1.766-2.669 0-1.602-.781-2.916-2.053-2.916zm14.494 3.613c-.369.041-3.203.246-5.789.329.082 2.71 1.683 3.859 3.285 3.859.903 0 1.56-.246 2.381-1.067l.534.739c-1.355 1.683-2.957 2.094-3.696 2.094-2.915 0-4.599-2.217-4.599-4.845 0-3.203 2.423-5.708 4.845-5.708 2.012 0 3.613 1.725 3.613 3.655.001.615-.123.862-.574.944zm-3.49-3.572c-.944 0-1.93 1.027-2.176 2.792l3.531-.082c.452 0 .534-.123.534-.534-.001-1.109-.78-2.176-1.889-2.176zm11.825 1.519a1.402 1.402 0 00-.985-.411c-.534 0-1.232.616-1.683 1.889v3.819c0 1.396.164 1.52 1.929 1.643v.821h-5.625v-.821c1.437-.123 1.56-.247 1.56-1.643v-4.64c0-1.396-.082-1.396-1.396-1.643v-.698c1.314-.247 2.464-.493 3.531-.903v2.464h.041c.698-1.355 1.724-2.422 2.71-2.422.78 0 1.396.616 1.396 1.273 0 .943-1.026 1.765-1.478 1.272z" fill="#00285A" fill-rule="nonzero"/></g></svg>
```

Comment:
Should we be including the width and height attributes on this in light of the recent discussions about `1em` width and height being applied to `u-icon` (which I assume will be applied to this)?

Reply:
@benjclark This one should remain as is, I believe. It's not a `u-icon` component and it doesn't use `<use>`.